src/i18n/es.js

```diff
@@ -174,6 +174,8 @@ export default {
   reports_pageranking: {
     top_pages: `Páginas más visitadas`,
     total_visits: `Visitantes únicos`,
+    visits_with_email: `Visitas CON email`,
```

Comment:
Is this CON ok? in english is all lowercase...

Reply:
i changed